### ISSUE TITLE
Allow `batch-mark-files-as-viewed`'s "toggle all with alt-click" on checkbox label

### ISSUE THREAD

**Issue by FloEdelmann** [sindresorhus/refined-github#4155]
In #4016, @yakov116 added the "toggle all with alt-click" to `batch-mark-files-as-viewed`, but it works only if you click directly on the checkbox (`.js-reviewed-checkbox`).

It'd be nice if it would also work when clicking on the whole label (`.js-reviewed-toggle`). A click or alt-click does trigger this one checkbox to change, but not all others.

![grafik](https://user-images.githubusercontent.com/202916/112480982-d7bbd280-8d76-11eb-91dc-a876f3bc523e.png)

**Comment by fregante** [comment]
It should probably be js-toggle-user-reviewed-file-form like the other 2

**Comment by FloEdelmann** [comment]
`.js-toggle-user-reviewed-file-form` looks too big:

![grafik](https://user-images.githubusercontent.com/202916/112690704-8564d980-8e7c-11eb-8905-2f5a9d9eab1c.png)

`.js-reviewed-toggle` is exactly the clickable area:

![grafik](https://user-images.githubusercontent.com/202916/112690754-9dd4f400-8e7c-11eb-978e-bf8b82d060d7.png)

The existing shift-click logic should also be changed to that IMO.

**Comment by fregante** [comment]
Indeed, that logic used the form because we’re catching the submit event in one case.
